src/main/java/org/spongepowered/api/Client.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api;
+
+import org.spongepowered.api.entity.living.player.ClientPlayer;
+import org.spongepowered.api.network.ServerConnection;
+import org.spongepowered.api.util.ThreadContext;
+import org.spongepowered.api.world.World;
+
+import java.net.InetSocketAddress;
+import java.nio.file.Path;
+import java.util.Optional;
+
+/**
+ * Represents the Minecraft client.
+ */
+public interface Client extends ThreadContext {
+
+    /**
+     * Gets the currently loaded world, if any.
+     *
+     * @return The current world
+     */
+    Optional<World> getLoadedWorld();
+
+    /**
+     * Gets the current client player entity, if it exists.
+     *
+     * @return The client player
+     */
+    Optional<ClientPlayer> getClientPlayer();
+
+    /**
+     * Gets the current local server instance if it is running.
+     *
+     * @return The local server
+     */
+    Optional<LocalServer> getLocalServer();
+
+    /**
+     * Loads a world save with the given name.
+     *
+     * @param saveLocation The location of the save
+     * @see Game#getSavesDirectory()
+     */
+    void loadWorldSave(Path saveLocation);
```

Comment:
There should be a way to know whether the loafing succeeded and the server/game is running